pkg/operator/ceph/object/user/controller.go

```diff
@@ -48,8 +49,9 @@ import (
 )
 
 const (
-	appName        = object.AppName
-	controllerName = "ceph-object-store-user-controller"
+	appName                          = object.AppName
+	controllerName                   = "ceph-object-store-user-controller"
+	updateObjectUserSecretAnnotation = "rook.io/source-of-truth"
```

Comment:
how to avoid this error
```
G101: Potential hardcoded credentials (gosec)
```

Reply:
https://github.com/rook/rook/blob/94953f3f7b81f678ee18c7d793d566f886bc978f/cmd/rook/ceph/osd.go#L83-L86